otx/algorithms/classification/adapters/mmcls/model/heads/selfsl_cls_head.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2022 Intel Corporation
```

Comment:
Right now we keep all the heads, losses, etc in the MPA repo. In the future this repo will be moved to otx anyway, but for now I'd keep all the custom modules together in one place.